static/js/compose_actions.js

```diff
@@ -182,6 +182,8 @@ function fill_in_opts_from_current_narrowed_view(msg_type, opts) {
         // Set default parameters based on the current narrowed view.
         ...narrow_state.set_compose_defaults(),
 
+        // Set default parameters based on provided opts, overwriting
```

Comment:
These are not default; these are the opposite of default.

Reply:
Right, updated. Thanks.